provider/replacements.json

```diff
@@ -1035,6 +1035,10 @@
     {
       "old": "* `inline_policy` - (Optional, **Deprecated**) Configuration block defining an exclusive set of IAM inline policies associated with the IAM role. See below. If no blocks are configured, Terraform will not manage any inline policies in this resource. Configuring one empty block (i.e., `inline_policy {}`) will cause Terraform to remove _all_ inline policies added out of band on `apply`.\n",
       "new": "* `inline_policy` - (Optional, **Deprecated**) Configuration block defining an exclusive set of IAM inline policies associated with the IAM role. See below. If no blocks are configured, Pulumi will not manage any inline policies in this resource. Configuring one empty block (i.e., `inline_policy {}`) will cause Pulumi to remove _all_ inline policies added out of band on `apply`.\n"
+    },
+    {
+      "old": "* `managed_policy_arns` - (Optional, **Deprecated**) Set of exclusive IAM managed policy ARNs to attach to the IAM role. If this attribute is not configured, Terraform will ignore policy attachments to this resource. When configured, Terraform will align the role's managed policy attachments with this set by attaching or detaching managed policies. Configuring an empty set (i.e., `managed_policy_arns = []`) will cause Terraform to remove _all_ managed policy attachments.\n",
+      "new": "* `managed_policy_arns` - (Optional, **Deprecated**) Set of exclusive IAM managed policy ARNs to attach to the IAM role. If this attribute is not configured, Pulumi will ignore policy attachments to this resource. When configured, Pulumi will align the role's managed policy attachments with this set by attaching or detaching managed policies. Configuring an empty set (i.e., `managed_policy_arns = []`) will cause Pulumi to remove _all_ managed policy attachments.\n"
```

Comment:
Shouldn't we remove the deprecation here as well?

Reply:
I do not mind this one as much since it is not showing in the builds and the lift on scrubbing all the documentation is higher.

Reply:
But it will show in the docs, right? I feel like that will be a bit confusing to users (kind of marked as deprecated in the docs but not on the type side)

I don't think it's a big lift to scrub this, it's just two descriptions (managed policies and inline policies). Am I missing something?

Reply:
Do you think this is the only place where it shows in the docs? Let me have a look.

Reply:
I'm kind of in favor of keeping it in the docs so at least the users know of the possibility. But I also understand if the decision is to completely hide this from the user, in which case we should remove from the docs as well.